iOS/Flows/People/PeopleCoordinator.swift

```diff
@@ -9,19 +9,26 @@
 import Foundation
 import StreamChat
 import Contacts
+import ContactsUI
+import MessageUI
+import Localization
 
 class PeopleCoordinator: PresentableCoordinator<[Person]> {
-    lazy var peopleNavController = PeopleNavigationController()
-    
-    var messageComposer: MessageComposerViewController?
+
+    private lazy var peopleNavController = PeopleNavigationController()
+    private var messageController: MessageComposerViewController?
```

Comment:
People coordinator is all moved into one file and it now uses async/await throughout. Should be easier to follow the logic.